src/database/models/index.js

```diff
@@ -4,7 +4,13 @@ import Sequelize from "sequelize";
 import envConfigs from "../config/config";
 
 const basename = _basename(__filename);
-const env = process.env.NODE_ENV || "development";
+let env;
```

Comment:
line 7 was better  just have a `NODE_ENV`  variable in `.env` file and give it value of the environment you are in.  Because those conditions don't consider the `production` environment

Reply:
sure!